ts-starter-complete-cotta/scripts/main.ts

```diff
@@ -41,7 +49,7 @@ function initializeBreakTheTerracotta() {
   let players = world.getAllPlayers();
 
   for (let player of players) {
-    player.runCommand("scoreboard players set @s score 0");
+    scoreObjective.setScore(player.scoreboardIdentity as ScoreboardIdentity, 0);
```

Comment:
❤️

Reply:
Ok, minor comment, would be better to do an undefine check here to avoid using `as`

Reply:
Hm yeah, I was actually confused why/when scoreboardIdentity could be null but I guess if one of the players is a spectator. I'll throw in an undefined check